File: lessons/compiled_lang/cpp/src/g10.array.cpp
#include <iostream>
#include <string>

int main() {
    std::string nicknames[] = {"bob", "ed", "steve", "ralph", "joe", "deb", "kate"};

    std::cout << "The names are: " << std::endl;
    for (const auto& name : nicknames) {
        std::cout << "  " << name << std::endl;
    }

    return 0;
}

File: lessons/compiled_lang/cs/src/g10.array.cs
using System;

class G10Array
{
    static void Main()
    {
        string[] nicknames = { "bob", "ed", "steve", "ralph", "joe", "deb", "kate" };

        Console.WriteLine("The names are: ");
        foreach (string name in nicknames)
        {
            Console.WriteLine("  " + name);
        }
    }
}
